product_docs/docs/pem/9/pem_rel_notes/960_rel_notes.mdx

```diff
@@ -10,11 +10,11 @@ New features, enhancements, bug fixes, and other changes in PEM 9.6.0 include:
-| Enhancement  | Added two alert templates &mdash; `Inactive replication slots` and `Conflicting replication slots` and added two probes &mdash; `Replication slots` and `Multixact ID Exhaustion (Wraparound)`.                                          |
-| Enhancement  | The default display name for a newly registered agent is now the host's fully qualified domain name (FQDN), falling back to the previous default of the hostname if the FQDN is not set.                                                 |
+| Enhancement  | PEM now includes the error message in email notifications of a failed job. The default character limit for error output is 1000, which can be configured now using the `job_notification_error_limit` parameter in the server configuration. |
+| Enhancement  | Added two alert templates (`Inactive replication slots` and `Conflicting replication slots`) and two probes (`Replication slots` and `Multixact ID Exhaustion (Wraparound)`).                                          |
+| Enhancement  | The default display name for a newly registered agent is now the host's fully qualified domain name (FQDN), falling back to the previous default of the hostname if the FQDN isn't set.                                                 |
 | Enhancement  | Updated the alert template `Table size in server/database/schema` with more detailed SQL query.                                                                                                                                          |
-| Bug&nbsp;fix | Fixed an issue where audit manager was setting `edb_audit_statement` parameter to '' instead of 'none' when log statement parameter was left empty in the GUI resulting in failure of server restart.                                    |
-| Bug&nbsp;fix | Fixed an issue whereby an error "`NoneType` has no `len()`" would occur when alerts were listed on the alert dashboard.                                                                                                                  |
-| Bug&nbsp;fix | Fixed an issue where team role was not working with PEM agents as expected.                                                                                                                                                              |
-| Bug&nbsp;fix | Fixed an issue with SQL query for the PEM agent registration. Added the public schema name in the table prefix to execute the SQL query successfully even if schema name is not in the search path.                                      |
+| Bug&nbsp;fix | Fixed an issue where audit manager was setting `edb_audit_statement` parameter to '' instead of 'none' when the log statement parameter was left empty in the GUI. This resulted in the failure of server restart.                                    |
```

Comment:
```suggestion
| Bug&nbsp;fix | Fixed an issue where audit manager was setting `edb_audit_statement` parameter to '' instead of 'none' when the log statement parameter was left empty in the GUI. This issue resulted in the failure of server restart.                                    |
```